### PR TITLE
docs(xtac-starter): document external task client starter

### PR DESCRIPTION
**Pull request by tasso94**
related to CAM-13130

### CODE REVIEW

File: content/user-guide/ext-client/spring-boot-starter.md [hunk 1]
```diff
@@ -0,0 +1,586 @@
...
+        variable-names: defaultScore
+```
+
+Or, by defining configuration attributes in the annotation:
+
+```java
+@ExternalTaskSubscription(
+  topicName = "creditScoreChecker",
+  processDefinitionKey = "loan_process",
+  includeExtenstionProperties = true,
```
Comment (tmetzke) [inline content/user-guide/ext-client/spring-boot-starter.md:64]:
```suggestion
  includeExtensionProperties = true,
```

File: content/user-guide/spring-boot-integration/_index.md [hunk 2]
```diff
@@ -29,7 +29,11 @@ To enable Camunda Platform auto configuration, add the following dependency to y
 
 This will add the Camunda engine v.{{< minor-version >}}.0 to your dependencies.
 
-Other starters that can be used are: [`camunda-bpm-spring-boot-starter-rest`](rest-api) and [`camunda-bpm-spring-boot-starter-webapp`](webapps).
+Other starters that can be used are: 
+
+* [`camunda-bpm-spring-boot-starter-rest`](rest-api)
+* [`camunda-bpm-spring-boot-starter-webapp`](webapps)
+* [`camunda-bpm-spring-boot-starter-external-task-client`]({{< ref "user-guide/ext-client/spring-boot-starter.md" >}})
```
Comment (tmetzke) [inline content/user-guide/spring-boot-integration/_index.md:36]: I am not 100% sure, but I think there has to be a leading `/` here.

```suggestion
* [`camunda-bpm-spring-boot-starter-external-task-client`]({{< ref "/user-guide/ext-client/spring-boot-starter.md" >}})
```

**Review by tmetzke — APPROVED** [review]
That looks really nice, is quite extensive and nice to read! I like it a lot! 👍

File: content/user-guide/ext-client/spring-boot-starter.md (hunk 1 above)
Comment (tasso94) [reply content/user-guide/ext-client/spring-boot-starter.md]: 👍

File: content/user-guide/spring-boot-integration/_index.md (hunk 2 above)
Comment (tasso94) [reply content/user-guide/spring-boot-integration/_index.md]: 👍

**Pull request merged**